index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <div class="header">
+            <div class="logo">
+                <img src="My Logo.png" alt="" class="logo-img" />
+            </div>
+            <div class="logo"></div>
+        </div>
```

Comment:
- [ ] Kindly use semantic tags rather than using div tags. This part should be wrapped into a `header` tag. 
- [ ] I have noticed you didn't include the hamburger button. Kindly add it to a `nav` tag.